src/SFML/Window/iOS/WindowImplUIKit.hpp

```diff
@@ -72,7 +72,7 @@ class WindowImplUIKit : public WindowImpl
     /// \brief Destructor
     ///
     ////////////////////////////////////////////////////////////
-    ~WindowImplUIKit();
+    ~WindowImplUIKit() = default;
```

Comment:
Because this is not virtual, I believe we can remove it entirely

Reply:
Fixed